src/exchanges/bitfinex-client.js

```diff
@@ -270,6 +367,24 @@ class BitfinexClient extends BasicClient {
     const sequenceId = Number(msg[2]);
     const timestampMs = msg[3];
 
+    if (msg[1] === "hb") {
```

Comment:
Extract to a `_onLevel2Heartbeat` method and add method level comment with sample JSON